File: app/src/main/java/awais/instagrabber/repositories/responses/directmessages/DirectItemReactions.java
package awais.instagrabber.repositories.responses.directmessages;

import java.util.List;

public class DirectItemReactions {
    private final List<DirectItemEmojiReaction> emojis;
    private final List<DirectItemEmojiReaction> likes;

    public DirectItemReactions(final List<DirectItemEmojiReaction> emojis,
                               final List<DirectItemEmojiReaction> likes) {
        this.emojis = emojis;
        this.likes = likes;
    }

    public List<DirectItemEmojiReaction> getEmojis() {
        return emojis;
    }

    public List<DirectItemEmojiReaction> getLikes() {
        return likes;
    }
}

File: app/src/main/java/awais/instagrabber/repositories/responses/directmessages/DirectItemReactions.kt
package awais.instagrabber.repositories.responses.directmessages

import java.io.Serializable

data class DirectItemReactions(
    var emojis: List<DirectItemEmojiReaction>? = null,
    var likes: List<DirectItemEmojiReaction>? = null,
) : Cloneable, Serializable {
    @Throws(CloneNotSupportedException::class)
    public override fun clone(): Any {
        return super.clone()
    }
}